src/main/java/teammates/ui/controller/InstructorCommentsPageData.java

```diff
@@ -158,33 +154,36 @@ private void setCommentsForStudentsTables() {
         }
     }
     
-    private List<CommentRow> createCommentRows(String giverEmail, String giverName) {
+    private List<Comment> createCommentRows(String giverEmail, String giverName) {
         
-        List<CommentRow> rows = new ArrayList<CommentRow>();
+        List<Comment> rows = new ArrayList<Comment>();
         List<CommentAttributes> commentsForGiver = comments.get(giverEmail);
         for (int i = 0; i < commentsForGiver.size(); i++) {            
-            String recipientDetails = getRecipientNames(commentsForGiver.get(i).recipients);
-            String creationTime = 
-                    Const.SystemParams.COMMENTS_SIMPLE_DATE_FORMATTER.format(commentsForGiver.get(i).createdAt);          
+            CommentAttributes comment = commentsForGiver.get(i);
+            String recipientDetails = getRecipientNames(comment.recipients);
             Boolean isInstructorAllowedToModifyCommentInSection = commentModifyPermissions.get(giverEmail).get(i);
-            String typeOfPeopleCanViewComment = getTypeOfPeopleCanViewComment(commentsForGiver.get(i));
-            String editedAt = commentsForGiver.get(i).getEditedAtText(giverName.equals("Anonymous"));
-            String showCommentsTo = getShowCommentsToForComment(commentsForGiver.get(i));
-            String showGiverNameTo = getShowGiverNameToForComment(commentsForGiver.get(i));
-            String showRecipientNameTo = getShowRecipientNameToForComment(commentsForGiver.get(i));
```

Comment:
You can remove these three `getShow*ForComment` methods since they aren't used anymore